docs/setup/sso.md

```diff
@@ -84,18 +84,19 @@ Sign-in with Apple uses special non-standard for authenticating clients, which r
 ```yaml
 upstream_oauth2:
   providers:
-    - client_id: 01JAYS74TCG3BTWKADN5Q4518C
-      client_name: "<Service ID>" # TO BE FILLED
+    - id: 01JAYS74TCG3BTWKADN5Q4518C
+      issuer: "https://appleid.apple.com"
+      authorization_endpoint: "https://appleid.apple.com/auth/authorize"
+      token_endpoint: "https://appleid.apple.com/auth/token"
+      human_name: "Apple"
```

Comment:
We also have support for `brand_name: apple`, which will show the right Apple logo on the login option